tools/wpt/expectation.json

```diff
@@ -3150,5 +3150,8 @@
     "eventhandlers.any.html?wpt_flags=h2": false,
     "eventhandlers.any.html?wss": false,
     "referrer.any.html": true
+  },
+  "workers" : {
+    "WorkerNavigator-hardware-concurrency.any": true
```

Comment:
This test file doesn't exist.

Reply:
And `workers/WorkerNavigator-hardware-concurrency.any.html` will fail since #11338 isn't merged yet. And there doesn't seem to be any tests to test `hardwareConcurrenty` on the window's `Navigator` interface.

Reply:
Hmm, ok...I'll add a unit test for it